pypreprocess/nipype_preproc_spm_utils.py

```diff
@@ -148,6 +168,7 @@ def _do_subject_slice_timing(subject_data, TR, TA=None, spm_dir=None,
     assert 1 <= ref_slice <= nslices, ref_slice
 
     # compute slice indices / order
+    ref_slice -= 1
```

Comment:
Oh, here is the answer I assume ?

Reply:
I converted here the ref_slice form Matlab to Python index.
Indeed the following of the code expects a Python ref_slice starting at zero.